sdk/tests/deqp/functional/gles3/00_test_list.txt

```diff
@@ -27,9 +27,9 @@ clipping.html
 defaultvertexattribute.html
 fbocompleteness.html
 fbodepthbuffer.html
-fbomultisample.2_samples.html
-fbomultisample.4_samples.html
-fbomultisample.8_samples.html
+--min-version 2.0.1 fbomultisample.2_samples.html
```

Comment:
Let's not take this out to 2.0.1.

Instead, we should add checkFramebufferStatus() to all these optional multisampled float formats.  If it returns something other than FRAMEBUFFER_COMPLETE, skip further testing but still allow the test to pass.